src/components/dropdown/dropdown.tsx

```diff
@@ -33,6 +33,7 @@ interface DropdownProps {
   onFocus?: () => void;
   onBlur?: () => void;
   renderOption?: RenderDropdownOption;
+  isDropdownListExpanded?: boolean;
```

Comment:
I think it is not necessary to mention 'dropdown' here.
What do you think about `isListWidthLimited' or just 'limitedListWidth'?